startup.S

```diff
@@ -4,6 +4,7 @@
 /* Exception vector. */
 .section .exception
 .globl __reset_vector
+.org 0x0
```

Comment:
It explicitly forces the routine to be placed exactly at the beginning of the `.exception` section. This was unimportant before changes introduced by this branch, because there was only one chunk of code to be placed in that section, so it was okay to assume it is going to be placed there. Not using this directive should yield the same result, but I prefer to be explicit about placement within `.exception`, as offsets there have a special meaning and routines shall not be placed arbitrarily. So this directive simply underlines that this routine has to be at `0xbfc00000`.

In particular, this may save us from problems (by emitting warnings/errors) in case `intr.S` was processed before `startup.S` (without this directive `__reset_vector_` would just get appended after interrupt procedures, which is clearly wrong), and in the very unlikely case that the `__reset_vector_` does not fit before other routines placed at further offsets.